core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -80,11 +82,11 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
   import dataSource.dataAccess.driver.api._
 
   def CreateWorkspace(workspace: WorkspaceRequest, parentSpan: Span = null) = createWorkspace(workspace, parentSpan)
-  def GetWorkspace(workspaceName: WorkspaceName, params: WorkspaceFieldSpecs) = getWorkspace(workspaceName, params)
+  def GetWorkspace(workspaceName: WorkspaceName, params: WorkspaceFieldSpecs, parentSpan: Span = null) = getWorkspace(workspaceName, params, parentSpan)
```

Comment:
`null` is not very scala-y and gives us the heebie jeebies. can you type this as `Option[Span]` and use `None` instead?

Reply:
it was for backwards compatibility, but turns out there is only one caller so just removed it